CreateBootstrappedMSBuild.cmd

```diff
@@ -16,15 +15,14 @@ if not defined VS140COMNTOOLS (
 )
 
 if not "%1"=="" (
-    set OutputPath=%1
+    set AdditionalBuildCommand=%1
 )
 
 echo ** Building with the installed MSBuild
-echo ** Output Path: %DebugBuildOutputPath%
-echo ** Additional Build Parameters:%AdditionalBuildCommand%
+echo ** Additional Build Parameters: %AdditionalBuildCommand%
 echo.
 :: Build MSBuild
-call "%~dp0build.cmd" /t:Rebuild %AdditionalBuildCommand%
+call "%~dp0build.cmd" /t:Rebuild /p:Platform=x86 %AdditionalBuildCommand%
```

Comment:
Could you make this a parameter that gets pushed down here from RebuildWithLocalMSBuild? That way the script can be called for any architecture, though I don't know how relevant that requirement is.